### PR TITLE
Rename to include Local in GPU docker

### PR DESCRIPTION
**Pull request by sidhulyalkar**
@ttngu207 I am adding the event_support.py here in this PR as I had already committed it. Can you take a quick look to see if I should change anything?

### CODE REVIEW

**Comment by drewyangdev** [comment]
@ttngu207 I'll merge this for now to test the worker spawning process

File: workflow/support/event_support.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from workflow import db_prefix
+from workflow.pipeline import event, session
+from workflow.utils.paths import get_raw_root_data_dir
+from .file_manifest import FileManifest, support_db_prefix
+import datajoint as dj
+
+ global _wf_external_inbox_path, _wf_external_outbox_path
```
Comment (ttngu207) [inline workflow/support/event_support.py:7]: don't need `global` here

File: workflow/support/event_support.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    beh_files = ['events.csv','trial.csv','block.csv']
+
+
+
+    file_keys, files = (
+            FileManifest
+            & f"remote_fullpath LIKE '%{session_full_dir}/Behavior/{f}%'"
+            for f in beh_files
+            & key
```
Comment (ttngu207) [inline workflow/support/event_support.py:49]: don't need this `& key` here, it doesn't do anything - `FileManifest` has nothing related to session.Session (`key`) here.

File: workflow/support/event_support.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        ).fetch("KEY", "file")
```
Comment (ttngu207) [inline workflow/support/event_support.py:50]:
```
    file_keys, files = (
            FileManifest
            & [f"remote_fullpath LIKE '%{session_full_dir}/Behavior/{f}'"
            for f in ('events.csv', 'trial.csv', 'block.csv')]
        ).fetch("KEY", "file")
```

**Pull request merged**